spec/lib/mcb/commands/courses/sync_to_find_spec.rb

```diff
@@ -16,47 +16,54 @@ def sync_to_find(provider_code, *course_codes)
   let(:email) { "user@education.gov.uk" }
   let(:provider) { create(:provider, provider_code: provider_code) }
   let!(:course) { create(:course, provider: provider, course_code: course_code) }
-  let!(:manage_api_request) {
-    stub_request(:post, "#{Settings.manage_api.base_url}/api/Publish/internal/course/#{provider_code}/#{course_code}")
-      .with { |req| req.body == { "email": email }.to_json }
+  let(:body) do
```

Comment:
I think the test can be simplified like 

https://github.com/DFE-Digital/manage-courses-backend/blob/f4cedf7c693592147600f4131405b21be343eb9d/spec/requests/api/system/force_sync_spec.rb